src/elm/Auth.elm

```diff
@@ -354,7 +354,7 @@ viewLoginSteps isModal shared model loginStep =
                     ]
                 , viewFieldLabel shared.translators "auth.login.wordsMode.input" passphraseId
                 , div [ class "relative" ]
-                    [ textarea
+                    [ input
```

Comment:
Well, I'm not sure if it's a good change...

Here's the current behavior, all words are visible (word wrapping is enabled):
![image](https://user-images.githubusercontent.com/140053/99813699-10ddff00-2b59-11eb-9f57-aa4c581aacea.png)

But with this change, we have a one-line input element without word wrapping:
![image](https://user-images.githubusercontent.com/140053/99813780-294e1980-2b59-11eb-99c7-9c6cd9df0d27.png)

And I'm even not sure if we can make `<input type="text">` to be multiline in HTML...

Maybe we can try to make this `input` to be multiline with some CSS/HTML properties. If not — I think it's better to have `textarea` here and use the `Enter` key handler to fire the form submit.

Reply:
I don't remember if that task asked for that.  It was better as it was, as @amiskov  commented.